src/statistics.cpp

```diff
@@ -207,6 +207,34 @@ static config write_by_cth_map(const std::map<int, struct statistics::stats::by_
 	return res;
 }
 
+static void merge_battle_result_maps(stats::battle_result_map& a, const stats::battle_result_map& b);
+static std::map<int, struct statistics::stats::by_cth_t> read_by_cth_map_from_battle_result_maps(const statistics::stats::battle_result_map& attacks, const statistics::stats::battle_result_map& defends)
+{
+	std::map<int, struct statistics::stats::by_cth_t> m;
+
+	statistics::stats::battle_result_map merged = attacks;
+	merge_battle_result_maps(merged, defends);
+
+	for(const auto& i : merged) {
+		int cth = i.first;
+		const statistics::stats::battle_sequence_frequency_map& frequency_map = i.second;
+		for(const auto& j : frequency_map) {
+			const std::string& res = j.first; // see attack_context::~attack_context()
+			const int occurrences = j.second;
+			unsigned int misses = std::count(res.begin(), res.end(), '0');
+			unsigned int hits = std::count(res.begin(), res.end(), '1');
+			if(misses + hits == 0)
```

Comment:
Please use braces.

Reply:
Done in 14a7139c018af0a5ccea6a110ab213a4ff6b17a7 along with the other smaller changes.